packages/sdk/CONTRIBUTING.md

```diff
@@ -60,11 +59,11 @@ Use `npm run format` to fix format issues and `npm run lint` to check lint issue
 
 1. Check out a new branch from "main".
 1. Update code in correct place. [Supporting Browser and NodeJS](#supporting-browser-and-nodejs)
-2. Make sure modified codes are covered by unit tests. [Running test cases](#running-test-cases)
-3. Ensure code style check has no error. [Style Guidelines](#style-guidelines)
-4. Add comment for public class/method. Please check [comment template](API_COMMENT.md) for details.
-5. Merge your changes to "main" branch.
-6. At least one approve from code owners is required.
+1. Make sure modified codes are covered by unit tests. [Running test cases](#running-test-cases)
+1. Ensure code style check has no error. [Style Guidelines](#style-guidelines)
+1. Add comment for public class/method. Please check [comment template](API_COMMENT.md) for details.
+1. Merge your changes to "main" branch.
+1. At least one approve from code owners is required.
```

Comment:
why not 1,2,3,4?

Reply:
Though 1,1,1,1 works well, other markdown in our repo (as well as other ms repos) use 1,2,3,4, may be we can align on this?

Reply:
fixed.

Reply:
Personal preference: 1 1 1 1